payments/repo.go

```diff
@@ -35,14 +35,64 @@ func (p *PaymentRepo) GetAllByBudgetID(id uint) ([]database.Payment, error) {
 func (p *PaymentRepo) GetAllByBudgetCategoryID(id uint) ([]database.Payment, error) {
 	var payments []database.Payment
 
-	err := p.db.Preload("BudgetCategory.Category").
+	err := p.db.
+		Preload("BudgetCategory.Category").
+		Preload("BudgetCategory.Category.User").
 		Where("budget_category_id = ?", id).
 		Order("date desc").
 		Find(&payments).Error
 
 	return payments, err
 }
 
+func (p *PaymentRepo) GetAllByCategoryID(id uint) ([]database.Payment, error) {
+	var payments []database.Payment
+
+	// TODO: Couldn't figure out how to properly filter by Category.ID, redo
```

Comment:
```.Where("something.field = ?", id)```

Reply:
categories.user_id is not a column, unless we join the table. And I have no idea how to use GORM Joins yet :D